src/main/java/org/gridsuite/modification/server/NetworkModificationController.java

```diff
@@ -269,4 +269,15 @@ public ResponseEntity<Void> deleteStashedModificationInGroup(@Parameter(descript
     public ResponseEntity<List<ModificationMetadata>> getModificationsMetadata(@RequestParam("ids") List<UUID> ids) {
         return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(networkModificationService.getModificationsMetadata(ids));
     }
+
+    @PutMapping(value = "/network-modifications/uuids", consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
it's weird to have "uuids" here ...

Why not "/network-modifications/apply" ?

To be confirmed with Slimane ?

Reply:
Renamed into : "/networks/{networkUuid}/apply"